src/ddspdrum/module.py

```diff
@@ -239,6 +239,38 @@ def make_argument(self, control_as_frequency: np.array):
         up_sampled = self.control_to_sample_rate(control_as_frequency)
         return np.cumsum(2 * np.pi * up_sampled / SAMPLE_RATE)
 
+    def oscillator(self, argument):
+        return np.cos(argument)
+
+
+class SquareSaw(VCO):
```

Comment:
Inherits from VCO (might be a better way to do this, but it's pretty smooth).

Reply:
Let's call it SquareSawVCO? I'm not so into Polish variable names but I think it makes sense

Reply:
hah. I mean, if you like. It's getting a bit long as a name at that point. I'll make the change for now.